albyhub/docker-compose.yml

```
@@ -0,0 +1,22 @@
+version: "3.7"
+services:
+  app_proxy:
+    environment:
+      APP_HOST: albyhub_server_1
+      APP_PORT: 8080
+      PROXY_AUTH_ADD: "false"
+  server:
+    image: ghcr.io/getalby/hub:v1.6.0@sha256:45ec367860a8f42c6f66317dd5889d8e2af8b0df1e0bc59b33e67c08770e4e17
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
+      - ${APP_LIGHTNING_NODE_DATA_DIR}:/lnd:ro
+    environment:
+      LN_BACKEND_TYPE: "LND"
+      LND_ADDRESS: $APP_LIGHTNING_NODE_IP:$APP_LIGHTNING_NODE_GRPC_PORT
+      LND_CERT_FILE: "/lnd/tls.cert"
+      LND_MACAROON_FILE: "/lnd/data/chain/bitcoin/$APP_BITCOIN_NETWORK/admin.macaroon"
+      WORK_DIR: "/data/albyhub"
+      COOKIE_SECRET: "742375a0f444d6de9c692aa83ac43c10"
```

If you want this to be unique to a user, you can utilize `${APP_SEED}` here.
```suggestion
      COOKIE_SECRET: ${APP_SEED}
```

APP_SEED is a unique 256 bit long hex string (128 bits of entropy) deterministically derived from a user's umbrelOS seed and your app's ID.